sal_interfaces/MTM2/MTM2_Commands.xml

```diff
@@ -101,4 +101,46 @@
     </item>
 </SALCommand>
 
+<SALCommand>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_command_switchForceBalanceSystem</EFDB_Topic>
+    <Alias>switchForceBalanceSystem</Alias>
+    <Description>Switch the force balance (FB) system.</Description>
+    <item>
+        <EFDB_Name>status</EFDB_Name>
+        <Description>0: turn off FB. 1: turn on FB.</Description>
+        <IDL_Type>int</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+</SALCommand>
+
+<SALCommand>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_command_useZenithAngleFromMount</EFDB_Topic>
+    <Alias>useZenithAngleFromMount</Alias>
+    <Description>Use the zenith angle from main telescope mount (MTMount) control system.</Description>
+    <item>
+        <EFDB_Name>value</EFDB_Name>
+        <Description>This value is ignored.</Description>
+        <IDL_Type>boolean</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+</SALCommand>
+
+<SALCommand>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_command_setTemperatureOrigin</EFDB_Topic>
+    <Alias>setTemperatureOrigin</Alias>
+    <Description>Set the origin (offset) of temperature sensors.</Description>
+    <item>
+        <EFDB_Name>origin</EFDB_Name>
```

Comment:
I think this attribute would be better named as `temperature` instead of `origin`...

Reply:
Based on the document of M2 wish list, there should be only one offset value to apply to all temperature sensors. Can you help to check your document and decide you want single offset value or 16 values?

Reply:
If we use the name of `MTM2_command_setTemperatureOffset`, the attribute name is better to use `offset`. However, it is a reserved word that can not be used.

Reply:
Updated in commit: 7f353f2. Thanks!

Reply:
use ringOffset, intakeOffset, exhaustOffset for attributes